index.html

```diff
@@ -462,10 +462,8 @@ <h3><dfn data-lt="JwsVerificationKey2020" data-dfn-type="dfn" id="JwsVerificatio
 
       <p>A JWK publicKey.</p>
 
-      <p class="issue">
-We have <a href="https://github.com/w3c/did-core/issues/240">an open issue</a> in did core to discuss security / 
-privacy implications of interactions between JWK and 
-other vocabulary defined in did core.
+      <p class="issue" data-number="240">
```

Comment:
@msporny I tested this before committing it. unfortunately, that issue number is for did core, and not this repo... so this fix breaks the link.

Reply:
:( -- I see, apologies for that. This is going to be an issue moving forward, we can explicitly link for now, but probably need to be able to cross-link in a more general way... perhaps by using respec's `xref` feature?